packages/@sanity/core/src/actions/graphql/deleteApiAction.js

```diff
@@ -0,0 +1,30 @@
+module.exports = async function deployApiActions(args, context) {
```

Comment:
Should the function be named `deleteApiAction` here?

Reply:
Whoops, thanks!